src/main/res/values-fr/strings.xml

```diff
@@ -65,7 +65,7 @@ Si le dispositif GPS signale des données inexactes (par exemple, la localisatio
     <!-- Activity Type -->
     <string name="activity_type_airplane">navigation aérienne</string>
     <string name="activity_type_atv">quad</string>
-    <string name="activity_type_biking">cyclisme</string>
```

Comment:
Please remove this change; translations are handled by https://hosted.weblate.org/engage/opentracks/